DashAI/back/dataloaders/classes/dataloader.py

```diff
@@ -0,0 +1,70 @@
+from abc import abstractmethod
+
+from datasets import ClassLabel
+
+from DashAI.back.config_object import ConfigObject
+
+
+class DataLoader(ConfigObject):
+    """
+    Abstract class with base methods for all data loaders
+    """
+
+    @abstractmethod
+    def load_data(self, dataset_path, url=None):
+        pass
```

Comment:
All abstract methods must raise NotImplementedError as a default behavior.

> _exception `NotImplementedError` [...] In user defined base classes, abstract methods should raise this exception when they require derived classes to override the method, or while the class is being developed to indicate that the real implementation still needs to be added._

Source:
- https://docs.python.org/3/library/exceptions.html#NotImplementedError

```suggestion
        raise NotImplementedError
```

Reply:
I didn't know about that, thanks for the correction.